components/server/src/container-module.ts

```diff
@@ -129,7 +129,7 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
     ).inSingletonScope();
 
     bind(ImageBuilderClientConfig).toDynamicValue(() => {
-        return { address: "image-builder:8080" }
+        return { address: process.env.IMAGE_BUILDER_SERVICE || "image-builder-mk3:8080" }
```

Comment:
This `process.env` call should really move to env, leaving us here with:
```
    bind(ImageBuilderClientConfig).toDynamicValue(ctx => {
        const env = ctx.container.get<Env>(Env);
        return { address: env.imageBuilderAddress }
    });
```

Reply:
done